vkcs/vpnaas/resource_site_connection.go

```diff
@@ -161,12 +165,51 @@ func ResourceSiteConnection() *schema.Resource {
 				ValidateDiagFunc: networking.ValidateSDN(),
 				Description:      "SDN to use for this resource. Must be one of following: \"neutron\", \"sprut\". Default value is project's default SDN.",
 			},
+			"traffic_selector_ep_merge": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"sdn"},
```

Comment:
this would not work for projects with sprut sdn only - users do not have to specify the only sdn

Reply:
For this argument I really need to know sdn, at the time of creation, I need to know the current SDN

Reply:
all new projects got sprut sdn only. snds are not very transparent for users. in most cases user should do nothing with sdn

so that mandatory specifying sdn argument looks worse than a strange error
and that error a user got if he specified the argument
despite our note

otherwise i see another prolbem - how to develop a tf file w/ vpn to work with any sdn?

Reply:
done, add computed: true for suppress diff in plan for neutron